checker/tests/run-pass/factorial.rs

```diff
@@ -8,19 +8,19 @@
 
 // MIRAI_FLAGS --diag=strict
 
-#[macro_use]
-extern crate mirai_annotations;
-
-fn fact(n: u8) -> u128 {
-    if n == 0 {
-        1
-    } else {
-        let n1fac = fact(n - 1);
-        verify!(n1fac <= std::u128::MAX / (n as u128)); //~ possible false verification condition
-        (n as u128) * n1fac
-    }
-}
+// #[macro_use]
```

Comment:
Left over? Or if this should stay commented out, perhaps add a TODO or note why?

Reply:
There is a new failure, which is probably expected, but there also seems to be a failure in the code for matching actual with expected output. I'll get to it one way or another.